cmd/verify/network/cmd.go

```diff
@@ -106,43 +106,42 @@ func init() {
 
 func run(cmd *cobra.Command, _ []string) {
 	r := rosa.NewRuntime().WithAWS().WithOCM()
+	var cluster *cmv1.Cluster
 	defer r.Cleanup()
 
-	if cmd.Flags().Changed(subnetIDsFlag) {
-		if cmd.Flags().Changed(clusterFlag) {
-			r.Reporter.Errorf("One and only one of %s or %s is required",
-				clusterFlag, subnetIDsFlag)
-			os.Exit(1)
-		}
+	if cmd.Flags().Changed(clusterFlag) {
+		cluster = r.FetchCluster()
 
-		region, err := aws.GetRegion(arguments.GetRegion())
-		if err != nil {
-			r.Reporter.Errorf("Error getting region: %v", err)
-			os.Exit(1)
-		}
-		args.region = region
-	} else if cmd.Flags().Changed(clusterFlag) {
-		if cmd.Flags().Changed(subnetIDsFlag) {
-			r.Reporter.Errorf("One and only one of %s or %s is required",
-				clusterFlag, subnetIDsFlag)
+		if cluster.Status() != nil && cluster.Status().State() != cmv1.ClusterStateReady {
+			r.Reporter.Errorf("Cluster state is not ready: %v", cluster.Status().State())
```

Comment:
Curious as to why the cluster has to be ready to run verify the network?

Reply:
There's nothing that truly impedes it, but when running from a cluster if it is not ready it will either run from the inflights instead of on demand or have already ran from the inflights.

Reply:
i removed all checks for cluster state